draft-ietf-homenet-front-end-naming-delegation.mkd

```diff
@@ -98,38 +98,55 @@ Infrastructure via a Distribution Master (DM).
+
+3. {{!RFC7368}} emphazes that the home network is subject to connectivity
+   disruptions with the ISP.
+   But, names used within the home MUST be resilient against such disruption.
+
+So a goal of this specification is to make the public names resolvable within
+both the home network and on the Internet, even when there are disruptions.
+
+This is achieved by having a device inside the home network that builds,
 publishes, and manages a Public Homenet Zone, thus providing bindings
 between public names, IP addresses, and other RR types.
 
 The management of the names can be a role that the Customer Premises
-Equipment (CPE) does. Other devices in the home network could fulfill
-this role e.g. a NAS server, but for simplicity, this document assumes
-the function is located on one of the CPE devices.
-
-A home network may have multiple CPEs. Since management of the Public
-Homenet Zone involves DNS specific mechanisms that cannot be distributed
-over multiple servers (primary server), when multiple nodes can
-potentially manage the Public Homenet Zone, a single node needs to be
-selected per outsourced zone. This selected node is designated as
-providing the Homenet Naming Authority (HNA) function.
-
-The process by which a single HNA is selected per zone is not in scope
-for this document.
-
-CPEs, which may host the HNA function, as well as home network devices,
-are usually low powered devices not designed for terminating heavy
-traffic. As a result, hosting an authoritative DNS service visible to
-the Internet may expose the home network to resource exhaustion and
-other attacks. Additionally, the names could become unavailable during
-disruptions of the upstream Internet connectivity.
+Equipment (CPE) does.
+Other devices in the home network could fulfill this role e.g. a NAS server,
+but for simplicity, this document assumes the function is located on one of the CPE devices.
+
+The homenet architecture {{!RFC7368}} makes it clear that a home network may have multiple CPEs.
+The management of the Public Homenet Zone involves DNS specific mechanisms
+that cannot be distributed over multiple servers (primary server), when
+multiple nodes can potentially manage the Public Homenet Zone, a single node
+needs to be selected per outsourced zone.
+This selected node is designated as providing the Homenet Naming Authority (HNA) function.
+
+The process by which a single HNA is selected per zone is not in scope for this document.
+It is envisioned that a future document will describe an HNCP mechanism to elect the single HNA.
+
+CPEs, which may host the HNA function, as well as home network devices, are usually low powered devices not designed for terminating heavy traffic.
+As a result, hosting an authoritative DNS service visible to the Internet may expose the home network to resource exhaustion and other attacks.
+Additionally, the names could become unavailable during disruptions of the upstream Internet connectivity.
 
```

Comment:
The sentence "Additionally ... " is not clear to me, I think that we want to say that the hosting the HNA locally would enable internal communications even in the case of upstream internet connectivity.

Reply:
On the other hand, if the only copy of the public zone is on the Internet,
then Internet connectivity disruptions would make the names unavailable
inside the homenet.